components/automate-chef-io/content/docs/backup.md

```diff
@@ -191,11 +234,107 @@ For backups stored in an Amazon S3 bucket, use:
 chef-automate backup list s3://bucket_name/base_path
 ```
 
-where `bucket_name` is the name of the S3 bucket and `base_path` is an optional path within the bucket where the backups live.
+For backups stored in a GCS bucket, use:
+
+```shell
+chef-automate backup list gs://bucket_name/base_path
+```
+
+where `bucket_name` is the name of the S3 bucket `base_path` an optional path within the bucket where the backups live.
+
+## Restore Chef Automate
```

Comment:
I am moving this Restore section to the [Restore page](https://automate.chef.io/docs/restore/) in the Automate docs.